cookbooks/bcpc-hadoop/recipes/namenode_standby.rb

```diff
@@ -84,6 +84,14 @@
   end
 end
 
+if node[:bcpc][:hadoop][:hdfs][:ldap][:integration] == true
+  file "/etc/hadoop/conf/ldap-conn-pass.txt" do
+    content node[:bcpc][:hadoop][:hdfs][:ldap][:password]
```

Comment:
Since this password can come from an external entity, we will have to manually `chef-vault` it.